vdr/doc-creator.go

```diff
@@ -91,6 +91,7 @@ func (n NutsDocCreator) Create() (*did.Document, error) {
 		Controller:         []did.DID{*didID},
 		VerificationMethod: []*did.VerificationMethod{verificationMethod},
 		Authentication:     []did.VerificationRelationship{{VerificationMethod: verificationMethod}},
+		AssertionMethod:    []did.VerificationRelationship{{VerificationMethod: verificationMethod}},
```

Comment:
Is this really a good default thing to do? We need the authentication verification method to be there (because of spec), but the assertionMethod not (per-spec)

Reply:
will remove for now.